capstone/src/main/webapp/index.html

```diff
@@ -1,10 +1,44 @@
-<!DOCTYPE html>
-<html>
+
+<!doctype html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>Default Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link href="style.css" type="text/css" rel="stylesheet">
+        <title>Home</title>
+
```

Comment:
Please remove the extra empty lines in this file. Makes sense to have one between the two major divs, but the rest seem unnecessary

Reply:
I just did this. Let me know what else I need to do.